docs/tutorial/2_search_and_asdf_download.md

```diff
@@ -49,41 +58,82 @@ Because we only specified one attr, and it was unique to the dkist client (it st
 If we only want VBI datasets, that are unembargoed, between a specific time range we can use multiple attrs:
 
 ```{code-cell} python
-Fido.search(a.Time("2023/10/16 18:45", "2023/10/16 18:48") & a.Instrument.vbi & a.dkist.Embargoed(False))
+Fido.search(a.Time("2022-06-02 17:00:00", "2022-06-02 18:00:00") & a.Instrument.vbi & a.dkist.Embargoed(False))
 ```
 
-Note how the `a.Time` and `a.Instrument` attrs are not prefixed with `dkist`.
-These are general attrs which can be used to search multiple clients.
+Note how the `a.Time` and `a.Instrument` attrs are not prefixed with `dkist` - these are general attrs which can be used to search multiple sources of data.
 
-So far the returned results have had to match all the attrs provided, because we have used the `&` (logical and) operator to join them.
+So far all returned results have had to match all the attrs provided, because we have used the `&` (logical AND) operator to join them.
 If we want results that match either one of multiple options we can use the `|` operator.
-Let's also restrict our search to a particular proposal, `pid_2_114`.
 
 ```{code-cell} python
-res = Fido.search((a.Instrument.vbi | a.Instrument.visp) & a.dkist.Embargoed(False) & a.dkist.Proposal("pid_2_114"))
-res
+---
+tags: [output_scroll]
+---
+Fido.search((a.Instrument.vbi | a.Instrument.visp) & a.dkist.Embargoed(False))
 ```
 
-As you can see this has returned two separate tables, one for VBI and one for VISP, even though in fact the VBI table is empty.
+As you can see this has returned two separate tables, one for VBI and one for VISP.
+
+Because `Fido` can search other clients as well as the DKIST you can make a more complex query which will search for VISP data and context images from AIA at the same time:
```

Comment:
```suggestion
Because `Fido` can search other clients in addition to the DKIST specific client. You can create more complex queries which will search for VISP data and context images from AIA at the same time:
```